Source/Fuse.Maps/MapOverlay.uno

```diff
@@ -115,6 +115,16 @@ namespace Fuse.Controls
 		static Selector _centerLatitudeName = "CenterLatitude";
 		static Selector _centerLongitudeName = "CenterLongitude";
 		static Selector _radiusName = "Radius";
+		static int UID_POOL = 0;
```

Comment:
Code-style: Private fields should start with an underscore followed by a lower-case letter (`_uidPool`).

Reply:
fixed